ha/monitor/k8s/object_monitor.py

```diff
@@ -152,7 +161,8 @@ def run(self):
                     break
 
                 Log.debug(f"Received event {an_event}")
-                alert, event = EventParser.parse(self._object, an_event, self._object_state)
+                alert, event = EventParser.parse(self._object, an_event, self._object_state, \
+                    self._kwargs.get('resource_id_map', None))
```

Comment:
will parser always receive resource_id_map ? Will resource_id_mapalways be populated or can it be empty ?   What happens in case of backward compatibility ? is it possible that FQDNs are not populated in cluster.conf and so resource_id_map is empty ?